Lua/Comments.tmPreferences

```diff
@@ -15,6 +15,18 @@
 				<key>value</key>
 				<string>-- </string>
 			</dict>
+			<dict>
+				<key>name</key>
+				<string>TM_COMMENT_START_2</string>
+				<key>value</key>
+				<string>--[[</string>
+			</dict>
+			<dict>
+				<key>name</key>
+				<string>TM_COMMENT_END_2</string>
+				<key>value</key>
+				<string>--]]</string>
```

Comment:
A good point, but there are people that for some reason don't run color schemes (a reasonably small percentage though).  
I'm not opposed to either version, it's just the one I'm used to.